<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgHideWifiWpa</class>
 <widget class="QDialog" name="DlgHideWifiWpa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to Hidden Wi-Fi Network</string>
  </property>
  <widget class="QLabel" name="lbNetName">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>151</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbSecurity">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>106</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>280</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPwd">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>203</y>
     <width>18</width>
     <height>9</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbPassword">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>196</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxSecurity">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>100</y>
     <width>182</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>191</y>
     <width>182</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>280</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNetName">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>145</y>
     <width>182</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lbBoder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>327</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLeftupTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>240</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="lineUp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>412</width>
     <height>1</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="lineDown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>412</width>
     <height>1</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>lbBoder</zorder>
  <zorder>lePassword</zorder>
  <zorder>lbNetName</zorder>
  <zorder>lbSecurity</zorder>
  <zorder>btnCancel</zorder>
  <zorder>checkBoxPwd</zorder>
  <zorder>lbPassword</zorder>
  <zorder>cbxSecurity</zorder>
  <zorder>btnConnect</zorder>
  <zorder>leNetName</zorder>
  <zorder>lbLeftupTitle</zorder>
  <zorder>lineUp</zorder>
  <zorder>lineDown</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
